### PR TITLE
Correct spelling/grammar in tutorial 05

### PR DESCRIPTION
**Pull request by hendrikmakait**

### CODE REVIEW

File: notes/05_val_reg/basis.tex [hunk 1]
```diff
@@ -120,7 +120,7 @@ \subsection{Preprocessing: Sphering}
 \end{align}
 
 \textbf{Important:} Reuse the same pre-processing (centering, sphering) on validation set. \notesonly{Do not recompute a separate mean or sphering transformation for the validation set.}
-Also applies to test-set in case of a 3-way split.
+Also apply to test-set in case of a 3-way split.
```
Comment (kashefy) [inline notes/05_val_reg/basis.tex:123]: Also apply it to...

File: notes/05_val_reg/validation.tex [hunk 2]
```diff
@@ -398,7 +398,7 @@ \subsection{Improving how we estimate $\widehat E^G$}
 \question{Why should we care about how we estimate $\widehat E^G$?}
 
 \notesonly{
-- It's simply to identify potential overfitting or underfitting. Are we memorizing the training data or did we underestimate the complexity of function we're trying to fit? Once we identify what kind of problem we have with our model it implies which ``fixes'' are relevant and which would simply not help at all.
+- It's simple to identify potential overfitting or underfitting. Are we memorizing the training data or did we underestimate the complexity of function we're trying to fit? Once we identify what kind of problem we have with our model it implies which ``fixes'' are relevant and which would simply not help at all.
```
Comment (kashefy) [inline notes/05_val_reg/validation.tex]: Should stay simply, it refers to the reason not the difficulty of identitying

File: notes/05_val_reg/validation.tex [hunk 3]
```diff
@@ -324,9 +324,9 @@ \subsubsection{Different views of the bias-variance tradeoff}
...
+The above ``fixes'' rely on having concluded that the model is either overfitting or underfitting. And that this can potentially be mitigated by taking these different measures. However, it is important to understand how overfitting and underfitting reflect where we are on the bias-variance tradeoff. This means that repeatedly applying one solution to an overfitting model can eventually cause the model to underfit and vice-versa. One should also note that if one of the above doesn't work. It does not conclude that the model was not overfitting or underfitting. The above are simply guidelines on how to understand why a model has poor generalization performance and come up with simple techniques for finding a different bias-variance tradeoff.
```
Comment (kashefy) [inline notes/05_val_reg/validation.tex:327]: What changes here?

File: notes/05_val_reg/validation.tex [hunk 4]
```diff
@@ -324,9 +324,9 @@ \subsubsection{Different views of the bias-variance tradeoff}
...
 
-Although we've just talked about how increasing the size of the training set can help mitigate overfitting. Observing how $E^T$ and $\widehat E^G$ behave as a function of the training set size $p$ is a way of determining if an overfitting problem even exists. Keep in mind that $\widehat E^G$ would not directly depend on the size of training set as it is only evaluated on the hold-out set, which we won't change in size. Nevertheless, eveytime with train our nodel with a larger training set, we evaluate both $E^T$ and $\widehat E^G$ for this trained model.
+Although we've just talked about how increasing the size of the training set can help mitigate overfitting. Observing how $E^T$ and $\widehat E^G$ behave as a function of the training set size $p$ is a way of determining if an overfitting problem even exists. Keep in mind that $\widehat E^G$ would not directly depend on the size of the training set as it is only evaluated on the hold-out set, which we won't change in size. Nevertheless, every time we train our model with a larger training set, we evaluate both $E^T$ and $\widehat E^G$ for this trained model.
```
Comment (kashefy) [inline notes/05_val_reg/validation.tex:329]: What changes here?

**Comment by hendrikmakait** [comment]
Is there anything stopping this PR from being merged?

**Comment by kashefy** [comment]
I commented on several changes. Aren't they showing up on your end? The "simply" should not be changed and there were too large blocks marked as changed but I can't tell what exactly changed in them.

File: notes/05_val_reg/validation.tex [hunk 5]
```diff
@@ -324,9 +324,9 @@ \subsubsection{Different views of the bias-variance tradeoff}
 \notesonly{
 \textbf{Remember: it's always a bias-variance \underline{tradeoff}}\\
 
-The above ``fixes'' rely on having concluded that the model is either overfitting or underfitting. And that this can potentially be mitigated by taking these different measures. However it is important to understand how overfitting and underfitting reflect where we are on the bias-variance tradeoff. This means that repeatedly applying one solution to an  overfitting model can eventually cause the model to underfit and vice-versa. One should also note that if one of the above doesn't work. It does not conclude that the model was not overfitting or underfitting. The above are simply guidelines on how to understand why a model has poor generalization performance and come up with simple techniques for finding a different bias-variance tradeoff.
```
Comment (hendrikmakait) [inline notes/05_val_reg/validation.tex:327]: "However it is..." -> "However, it is..."

"applying one solution to an  overfitting model" -> "applying one solution to an overfitting model" (removed duplicates whitespace)

File: notes/05_val_reg/validation.tex [hunk 6]
```diff
@@ -324,9 +324,9 @@ \subsubsection{Different views of the bias-variance tradeoff}
...
 
-Although we've just talked about how increasing the size of the training set can help mitigate overfitting. Observing how $E^T$ and $\widehat E^G$ behave as a function of the training set size $p$ is a way of determining if an overfitting problem even exists. Keep in mind that $\widehat E^G$ would not directly depend on the size of training set as it is only evaluated on the hold-out set, which we won't change in size. Nevertheless, eveytime with train our nodel with a larger training set, we evaluate both $E^T$ and $\widehat E^G$ for this trained model.
```
Comment (hendrikmakait) [inline notes/05_val_reg/validation.tex:329]: I did a quick text diff on this line to highlight changes: https://www.diffchecker.com/4jfic4HH

**Comment by hendrikmakait** [comment]
Oddly enough the comments did not show up for me. I hope this helps.

**Review by kashefy — APPROVED** [review]

**Pull request merged**